tests/__init__.py

```diff
@@ -34,3 +36,14 @@ def clean():
     testcase.addCleanup(clean)
 
     return tempfile
+
+
+@nottest
+def generate_test_file(testcase, lines):
```

Comment:
Let's keep the previous naming style and call this a "temp" file. Maybe calling the function `create_tempfile`? A nice side effect is that you will not need the ugly `@nottest`

Reply:
create_tempfile sounds good to me.